pkg/plugins/autodiscovery/cargo/dependencyManifest.go

```diff
@@ -40,17 +40,83 @@ sources:
       Key: '{{ .ExistingSourceKey }}'
 conditions:
   {{ .ConditionID }}:
-    name: 'Ensure Cargo chart named "{{ .DependencyName }}" is specified'
+    name: 'Test if version of "{{ .DependencyName }}" {{"{{"}} source "{{ .ExistingSourceID }}" {{"}}"}} differs from {{"{{"}} source "{{ .SourceID }}" {{"}}"}}'
+    kind: 'shell'
+    sourceid: '{{ .SourceID }}'
+{{- if .ScmID }}
+    scmid: '{{ .ScmID }}'
+{{- end }}
+    spec:
+      command: 'test {{"{{"}} source "{{ .ExistingSourceID }}" {{"}}"}} != '
+targets:
+  {{ .TargetID }}:
+    name: '{{ .TargetName }}'
+    kind: 'shell'
+{{- if .ScmID }}
+    scmid: '{{ .ScmID }}'
+{{- end }}
+    spec:
+      command: |
+        ARGS=""
+        if [ "$DRY_RUN" = "true" ]; then
+          ARGS="--dry-run"
+        fi
+        cargo upgrade $ARGS --manifest-path {{ .File }} --package {{ .DependencyName }}@{{"{{"}} source "{{ .SourceID }}" {{"}}"}}
```

Comment:
Is it `cargo upgrade` or `cargo upgrades`
```suggestion
        cargo upgrades $ARGS --manifest-path {{ .File }} --package {{ .DependencyName }}@{{"{{"}} source "{{ .SourceID }}" {{"}}"}}
```

Reply:
Thanks I just found the link.
I think it would be better to update the message to suggest installing 

* https://github.com/killercup/cargo-edit